src/interfaces.ts

```diff
@@ -0,0 +1,1156 @@
+/* eslint-disable */
```

Comment:
This seems to have been generated, maybe with quicktype ?
Can we not generate it as part of the build ?

Reply:
if we disable it we have around 100 errors thrown by standard-ts which will cause everything to fail even though it is valid ts

Reply:
My comment is not on the eslint-disable but more on the whole file

Reply:
we could generate the file pre-build and not commit it to git like we do to data?

Reply:
Yes I think it whould be better, because this also mean that we don't need to update it if the schema changes

Reply:
I will finish that in the morning as it's causing some issues right now and I need sleep